lib/pub_sub/message.rb

```diff
@@ -1,7 +1,7 @@
 module PubSub
   class Message
     def initialize(payload)
-      @payload = JSON.parse(payload)
+      @payload = JSON.parse(JSON.parse(payload)['Message'])
```

Comment:
I thought the same thing. I dont understand why this is required and why any of the existing pub sub notifications are working. See the test for an example of the payload https://github.com/westfieldlabs/pub_sub/pull/20/files#diff-80a4462fe6bd52387957fb0a78b366a2R18